unit-tests/src/test/java/org/eclipse/collections/impl/map/sorted/mutable/UnmodifiableTreeMapTest.java

```diff
@@ -319,6 +319,12 @@ public void add()
     }
 
     @Override
+    public void putPair()
```

Comment:
Change this to use `@Test(expected = UnsupportedOperationException.class)` like the `withKeyValue` right below it.

Reply:
why? this is similar to `add()` above.

Reply:
Good point.  I would recommend changing `add` as well.  There are a much larger number of `@Test(expected =)` usages in this file.  I would fix all the others in this file to be consistent.  This approach is much simpler and easier to read.